lib/ansible/plugins/connection/network_cli.py

```diff
@@ -84,6 +88,9 @@ def _connect(self):
 
         display.display('ssh connection done, setting terminal', log_only=True)
 
+        self._shell = self.ssh.invoke_shell()
```

Comment:
Why opening the shell on the constructor and not opening it on the 'open_shell' method?